src/Domain/Entities/User.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Domain.Common;
+
+namespace Domain.Entities
+{
+    class User : BaseEntity
+    {
+        public string Name { get; set; }
+        public string Email { get; set; }
+        public bool EmailVerified { get; set; }
+        public string Role { get; set; }
+        public string Gender { get; set; }
+        public DateTime DateOfBirth { get; set; }
+        public string MobileNo { get; set; }
+    }
```

Comment:
Please check if this field should be included or not @deba1  @kanismim 

- [ ]  `public string Address { get; set; }`
- [ ]  `public virtual ICollection<Feedback> Feedbacks { get; set; }`
- [ ]  `public virtual ICollection<Order> Orders { get; set; }`
`public virtual UserCredential UserCredential { get; set; }`